l1-contracts/upgrade-envs/outputs/stage-proofs/verification-logs

```diff
@@ -0,0 +1,31 @@
+forge verify-contract 0xD7Dc68e7e9ED4b6970c4D9a2D58591a622A24330 BytecodesSupplier
+forge verify-contract 0x1BB20182766A71f65333690abC84CAE4d1Dd4C4A Verifier
+forge verify-contract 0x2c50C89221fE5cE9B8b9e9784B10D03d0208F1DB DefaultUpgrade
```

Comment:
maybe it is not mentioned in the doc, but please use `--watch` flag, it will ensure that the contracts were actually verified